### ISSUE TITLE
WearOS mode 

### ISSUE THREAD

**Issue by srt54558** [simondankelmann/Bluetooth-LE-Spam#192]
Is there a way of implementing a functionality for wearOS? Like install the app and start spamming from the watch or remotely from the phone?

**Comment by GM3322** [comment]
Yeah that would be great, but only way to install the app on watch is by wireless adb (sideloading)

**Comment by srt54558** [comment]
> Yeah that would be great, but only way to install the app on watch is by wireless adb (sideloading)
That's not a problem and fairly easy

**Comment by ha2ked** [comment]
This will most likely not be added by us, but anyone is welcome to do so.
